routes/channel.js

```
@@ -35,6 +37,61 @@ function getStatus(req, res) {
 	res.send({ channel: req.channel })
 }
 
+function getEventTimeAggregate(req, res, next) {
+	const { publisher } = req.params
+	const {
+		eventType = 'IMPRESSION',
+		metric = 'eventCounts',
+		timeframe = 'year',
```

you should throw an error if the timeframe passed is invalid